src/main/java/org/sagebionetworks/bridge/hibernate/HibernateAccount.java

```diff
@@ -543,4 +544,16 @@ public String getNote() {
     public void setNote(String note) {
         this.note = note;
     }
+
+    /** The client's time zone explicitly set through requests and used to calculate
+     * schedules for the user. Can be updated or deleted. */
```

Comment:
Not used to calculate the schedule in the server, just for the client side as reference.